# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../efsd/efsd_automake.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -u -r -N efsd.orig/efsd/Makefile.am efsd/efsd/Makefile.am
--- efsd.orig/efsd/Makefile.am	2005-03-27 15:39:47.401970424 +0200
+++ efsd/efsd/Makefile.am	2005-03-27 15:40:42.217637176 +0200
@@ -7,7 +7,7 @@
 
 DEBUGFLAGS = -DDEBUG -DDEBUG_NEST -W -Wall
 
-INCLUDES = @edb_cflags@ @xml2_cflags@ $(DEBUGFLAGS)
+AM_CPPFLAGS = @edb_cflags@ @xml2_cflags@ $(DEBUGFLAGS)
 
 if USE_THREADS_FILES
 efsd_threadstuff = efsd_lock.c
@@ -44,6 +44,8 @@
 	efsd_types.h efsd_types.c 		\
 	$(efsd_threadstuff)
 
+efsd_CFLAGS = $(AM_CFLAGS)
+
 efsd_LDADD = @edb_libs@ @xml2_libs@
 
 lib_LTLIBRARIES = libefsd.la
diff -u -r -N efsd.orig/tools/Makefile.am efsd/tools/Makefile.am
--- efsd.orig/tools/Makefile.am	2005-03-27 15:39:47.409969208 +0200
+++ efsd/tools/Makefile.am	2005-03-27 15:41:02.481556592 +0200
@@ -5,7 +5,7 @@
 # A list of all the files in the current directory which can be regenerated
 MAINTAINERCLEANFILES = Makefile.in Makefile efsdsh
 
-INCLUDES = -I$(top_srcdir)/efsd
+AM_CPPFLAGS = -I$(top_srcdir)/efsd
 
 efsdsh_SOURCES = efsdsh.c
 efsdsh_LDADD = $(top_builddir)/efsd/.libs/libefsd.so
